modules/ROOT/nav.adoc

```diff
@@ -56,72 +54,6 @@
 **** xref:deploy:deployment-option/self-hosted/manual/sizing.adoc[Sizing Guidelines]
 **** xref:deploy:deployment-option/self-hosted/manual/linux-system-tuning.adoc[System Tuning]
 *** xref:deploy:deployment-option/self-hosted/docker-image.adoc[Connectors]
-** xref:deploy:deployment-option/cloud/index.adoc[Cloud]
```

Comment:
I notice we kept the 'Self Hosted' level in the nav tree. We can probably remove this now? Or if you want to keep Cloud and link to the Cloud docs, that would be good too.

** xref:deploy:deployment-option/self-hosted/index.adoc[Self-Hosted]

Reply:
Right, initially I removed that Self-Hosted level, but the redirects made my head spin. Is there any easy way to do that?